### PR TITLE
Accessibility Pair-Programmig

### PR DESCRIPTION
**Pull request by Mar12358**
# Checkpoints
## Image text alternatives: 
- [Changed alt text for images](https://github.com/Yakub-Ojo/My-Portfolio/commit/18d22e48b105f0299a76a13bac67c9f4b40fd3f7)
- [Fix repeated alternative texts](https://github.com/Yakub-Ojo/My-Portfolio/commit/dedb94729498523b8c42702c169f19bd293ccdaa)
## Text headings 
- [Change headings h2 and h3](https://github.com/Yakub-Ojo/My-Portfolio/commit/18d22e48b105f0299a76a13bac67c9f4b40fd3f7)
## Interaction 
- [Add invisible labels to form](https://github.com/Yakub-Ojo/My-Portfolio/commit/372aa8c5c763b00c350e3e6dee9366ed4061b71f)
- [Add links for menu list items](https://github.com/Yakub-Ojo/My-Portfolio/commit/ccf8fed4c2fd96747ca69d62a4ba46550c78cf81)
## The basic structure of the page
- [Move the nav-bar up](https://github.com/Yakub-Ojo/My-Portfolio/commit/43599cc62f5f97652a4c7dc60f9d954ac41f2eeb)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -239,27 +243,27 @@ <h3>Skills</h3>
...
           <li>
             <a href="#">
-              <img src="Icons/angellist.svg" alt="twitter">
+              <img src="Icons/angellist.svg" alt="angellist">
             </a>
           </li>
         </ul>
       </div>
-      <img class="ft-indicator" src="./Icons/Shape.svg" alt="shape">
+      <img class="ft-indicator" src="./Icons/Shape.svg" alt="">
```
Comment (joel-kalema) [inline index.html:266]: - [ ] Good job working on the accessibility 👍, but still need to to fix some points 😃. Kindly, make sure every image has `alt` with appropriate alternative text, that's useful for interacting or understanding the page has an `alt` text
- [ ] Also the line [76](https://github.com/Yakub-Ojo/My-Portfolio/pull/8/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L75:~:text=%3Cimg%20src%3D%22./Icons/Indicator.svg%22%20alt%3D%22%22%20/%3E) `` . Kindly, make sure every image has `alt` with appropriate alternative text, that's useful for interacting or understanding the page has an `alt` text 👍

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Mar12358 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medaminedev66** [comment]
Hello @Yakub-Ojo 👋,

_This is an additional review as a result of the second opinion request._

It's one of the project requirements that you should add image text alternatives for your images. Please check out the requirement below and work on the changes requested by the previous reviewer so that your project is complete.

image

Please take a look at [this article](https://www.w3.org/WAI/test-evaluate/preliminary/#images) to learn about the importance of using image alternatives.

Keep rocking 🚀!

**Review by PinkMoon25 — APPROVED** [review]
Hi @Yakub-Ojo  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linter checks :heavy_check_mark: 
github flow :heavy_check_mark: 
Accessibility :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**